packages/docs-site/.vitepress/config.ts

```diff
@@ -286,6 +288,10 @@ export default defineConfig({
       ],
     },
 
-    footer: { copyright: "made with ❤️ in Pittsburgh and abroad" },
+    footer: {
+      message:
+        'Released under the <a href="https://github.com/penrose/penrose/blob/main/LICENSE">MIT License</a>.',
```

Comment:
This link works but doesn't look like a link on the page. Is there a way to style it to look clickable?

Reply:
Not sure. I only followed the vitepress guide here: https://vitepress.dev/reference/default-theme-footer#footer for this one. Probably not high priority